### PR TITLE
Added the dynamic project feature 

### PR DESCRIPTION
**Pull request by ypratham**
Now, we can add new project in database and also delete them.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
...
+    ">>> **Help Commands** \n\nThese are the available commands:\n\n1. `!pdc help` - Dailogue of all commands\n2. `!pdc info` -  Gives info of bot\n3. `!pdc about` -  Returns server information\n4. `!pdc discord` - Provides invitation link for the discord server\n5. `!pdc github` - Provides link to the github organisation\n6. `!pdc core team` - Returns current Core Member\n7. `!pdc projects` - Returns active projects\n8. `!pdc quote`s - Returns random quote\n9. `!pdc events` - Returns upcoming events\n10. `!pdc new-event` - Add new event\n11. `!pdc delete-event` - Delete an event\n12. `!pdc list-events` - List all events\n13. `!pdc event-syntax` - List all syntax for events command\n14. `!pdc new-projec` - add new project to the list\n15. `!pdc delete-project` - delete a project from the list\n\n _Our bot is Open Source_"
```
Comment (cmulay) [inline main.py:42]: Here on line 42 `!pdc new project` should be there, you have missed the letter `'t`' at the end.
Also for the look at syntax: 
1. `!pdc new-project` - for creating new project 
`-` is missing in creating the new project !

**Review by cmulay — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (cmulay) [reply main.py:42]: There are few changes to be made before merging the PR!

Comment (ypratham) [reply main.py:42]: Okay, i will fix them right now

**Review by cmulay — APPROVED** [review]

**Comment by cmulay** [comment]
LGMT !

**Pull request merged**